app/src/main/java/net/measurementlab/ndt/android/MainActivity.java

```diff
@@ -0,0 +1,64 @@
+package net.measurementlab.ndt.android;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Toast;
+
+public class MainActivity extends AppCompatActivity {
+
+    @NonNull
+    private static final String TAG = "MainActivity";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        Settings settings = new Settings();
+        settings.hostname = "35.235.104.27";
+        settings.port = 443;
+        settings.disableTLS = false;
+        settings.skipTLSVerify = true;
+        settings.download.duration = 1;
+        settings.download.adaptive = true;
```

Comment:
These two settings (`duration` and `adaptive`) are not needed anymore because the protocol does not support them anymore. Can you remove them?